ui/tests/acceptance/job-run-test.js

```diff
@@ -158,9 +158,9 @@ module('Acceptance | job run', function (hooks) {
           fakeRequest.queryParams,
           {
             prefix: 'nomad/job-templates',
-            filter: 'Template is not empty"',
+            namespace: '*',
           },
-          'It makes a request to the /vars endpoint with the appropriate filter expression for job templates.'
+          'It makes a request to the /vars endpoint with the appropriate query parameters for job templates.'
         );
         return [];
       });
```

Comment:
Follow-up item request: add tests for "When variable created at nomad/job-templates/, shows up on the run/templates page", ditto delete, description change.

Reply:
This page isn't real-time. So we're really just testing given an input which is the URL, that we get a certain output and trigger the correct side-effect (dispatch a network request).

I can create some variables and ensure that those variables do not appear in the list, but that's already covered by ensuring that we dispatch a network request with the correct query parameters.

The tests you're describing sound like its to ensure that the Ember Data store isn't serving us a stale cache. So I'm a little confused about these specifications.

Reply:
Ember Data, the Ember Route and the Nomad API are implementation details and basically the blackbox of this view. But I can gladly test these details here if you think its best.